accessibility-oscilloscope/configs/qemu_defconfig

```diff
@@ -13,6 +14,7 @@ BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="board/qemu/arm-versatile/linux.config"
 BR2_LINUX_KERNEL_DTS_SUPPORT=y
 BR2_LINUX_KERNEL_INTREE_DTS_NAME="versatile-pb"
 BR2_PACKAGE_PYTHON3=y
+BR2_PACKAGE_LIBINPUT=y
```

Comment:
I think once you rebase this is the only change you'll need to replicate on pi4, eudev is already there for i2c stuff